test/e2e/apimachinery/flowcontrol.go

```diff
@@ -135,3 +299,48 @@ func testResponseHeaderMatches(f *framework.Framework, impersonatingUser, plUID,
 	}
 	return true
 }
+
+// uniformQPSLoadSingle loads the API server with requests at a uniform <qps>
+// for <loadDuration> time. The number of successfully completed requests is
+// returned.
+func uniformQPSLoadSingle(f *framework.Framework, username string, qps float64, loadDuration time.Duration) int32 {
+	var completed int32
+	var wg sync.WaitGroup
+	ticker := time.NewTicker(time.Duration(1e9/qps) * time.Nanosecond)
+	defer ticker.Stop()
+	timer := time.NewTimer(loadDuration)
+	for {
+		select {
+		case <-ticker.C:
+			wg.Add(1)
+			go func() {
+				makeRequest(f, username)
```

Comment:
Should we maybe consider latency? Like if a request takes 55 seconds to complete, should we count that? I guess I'd like comments to explain how it's supposed to interpret latency. Right now IIUC it reads as if we expect all requests to either instantly fail or succeed, which isn't how it'll actually work.

Reply:
Done.